xbmc/addons/PVRClient.cpp

```diff
@@ -1299,8 +1299,13 @@ bool CPVRClient::FillChannelStreamFileItem(CFileItem &fileItem)
   {
     if (strncmp(properties[i].strName, PVR_STREAM_PROPERTY_STREAMURL, strlen(PVR_STREAM_PROPERTY_STREAMURL)) == 0)
       fileItem.SetDynPath(properties[i].strValue);
-
-    fileItem.SetProperty(properties[i].strName, properties[i].strValue);
+    else if (strncmp(properties[i].strName, PVR_STREAM_PROPERTY_MIMETYPE, strlen(PVR_STREAM_PROPERTY_MIMETYPE)) == 0)
+    {
+      fileItem.SetMimeType(properties[i].strValue);
+      fileItem.SetContentLookup(false);
+    }
+    else
+      fileItem.SetProperty(properties[i].strName, properties[i].strValue);
```

Comment:
Why 'else'? I would also put the original property into the fileitem. That's what we do with "streamurl".

Reply:
do you want to have streamurl in properties?

Reply:
Not sure whether not changing min version min actually leads to no incompatibilities. @AlwinEsch ?

Reply:
And I see no reason against it. For me, it is a consistency thingy: Everything the addon sts as property can be found 1:1 at the FileItem.